src/pages/home/report/ReportActionCompose.js

```diff
@@ -307,7 +308,7 @@ class ReportActionCompose extends React.Component {
     updateComment(newComment) {
         this.textInput.setNativeProps({text: newComment});
         this.setState({
-            isCommentEmpty: newComment.length === 0,
+            isCommentEmpty: newComment.trim().length === 0,
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/21920. We used to relay on `isCommentEmpty` state being actually empty (length is 0) to move on to edit mode. This change made it possible to go into edit mode while having content on the Composer.

Reply:
@s77rt Could you elaborate on why that code causing the regression now?  The code got change on the next PR https://github.com/Expensify/App/pull/8844

Reply:
Sure! The edit mode was accessible based on `isCommentEmpty` state. Before if the composer only contains empty spaces `   ` and you keep pressing the arrow up key, nothing will happen because the comment is not empty. But now the comment length is based on the trimmed version of that text so the composer appears empty and pressing the arrow up will get you into edit mode.